CardanoSharp.Wallet/Extensions/Models/Transactions/IsValidExtensions.cs

```diff
@@ -0,0 +1,46 @@
+using CardanoSharp.Wallet.Models.Transactions;
+using PeterO.Cbor2;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+
+namespace CardanoSharp.Wallet.Extensions.Models.Transactions
+{
+    public static class IsValidExtensions
+    {
+        public static CBORObject GetCBOR(this bool isValid)
+        {
+            return CBORObject.FromObject(isValid);
```

Comment:
Converts to either CBORObject.True or CBORObject.False

https://github.com/peteroupc/CBOR/blob/b625bd32138c1f8cdd97fba728429d95d6b2de28/CBOR/PeterO/Cbor/CBORObject.cs#L2483